src/browser/api/window.js

```diff
@@ -502,6 +502,28 @@ Window.create = function(id, opts) {
             }
         });
 
+        const emitToAppAndWin = (...types) => {
+            let isMainWindow = (uuid === name);
+            types.forEach(type => {
+                ofEvents.emit(route.window(type, uuid, name), { topic: 'window', type, uuid, name });
+                if (isMainWindow) {
+                    ofEvents.emit(route.application(`window-${type}`, uuid), { topic: 'application', type, uuid, name });
```

Comment:
@datamadic @rdepena ok, hoping (865c44f) addresses all concerns. 😖  Notwithstanding, note that I left `name` off the event object for the additional Application event. *_Is this correct?_*